src/main/target/OMNIBUSF4V6/target.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of INAV.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Alternatively, the contents of this file may be used under the terms
+ * of the GNU General Public License Version 3, as described below:
+ *
+ * This file is free software: you may copy, redistribute and/or modify
+ * it under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+#include "drivers/pwm_mapping.h"
+#include "drivers/timer.h"
+#include "drivers/bus.h"
+
+#define DEF_TIM_CHNL_CH1    TIM_Channel_1
+#define DEF_TIM_CHNL_CH2    TIM_Channel_2
+#define DEF_TIM_CHNL_CH3    TIM_Channel_3
+#define DEF_TIM_CHNL_CH4    TIM_Channel_4
+
+#define DEF_TIM(_tim, _ch, _pin, _usage, _flags) \
+    { _tim, IO_TAG(_pin), DEF_TIM_CHNL_##_ch, _flags, IOCFG_AF_PP, GPIO_AF_##_tim, _usage }
+
+
+const timerHardware_t timerHardware[] = {
+    DEF_TIM(TIM10, CH1, PB8, TIM_USE_PPM,       0), // PPM
+
+    DEF_TIM(TIM3, CH3, PB0, TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR,   1), // S1_OUT
+    DEF_TIM(TIM3, CH4, PB1, TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR,   1), // S2_OUT
+    DEF_TIM(TIM9, CH2, PA3, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,   1), // S3_OUT
+    DEF_TIM(TIM3, CH2, PB5, TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,   1), // S4_OUT
+
+    DEF_TIM(TIM4, CH1, PB6, TIM_USE_LED,        0), // LED strip
```

Comment:
Is `PB6` also the pin for `I2C1_SCL`? If so does this matter?

Reply:
To be clear, this enhancement request is for the following board, yes?
https://store.myairbot.com/flight-controller/omnibus-f3-f4/omnibusf4v6.html
Has anyone tested this for sbus?  I used the fireworks target per instructions from airbot, my receiver and taranis setup are known good, board is calibrated, all icons green yet no activity in receiver tab of configurator when moving throttle, etc.   I am happy to help with SBUS and GPS specific testing (Beitan 880 GPS with I2C compass).   Documentation specific to this board would be nice also.  I can help with that as well if asked.

Reply:
Hi. This target, with the following PR, works with sbus. Fireworks will not work. I added the hex file a few months ago here, if you want to test it: 
https://www.dronetrest.com/t/i2c-magnetometer-not-working-on-omnibus-f4-v6/7005/11?u=fat
I have not had time to combine the targets.

Reply:
thanks I will try it!!!

Reply:
Uart 6 is Sbus, correct?
I must have a hardware issue, I flashed this in and still seen no response in the inav configurator to my arm switch or anything else.

Reply:
Hi, this is the same board. The AIO just means that the BEC is added (And I am not aware that there is another board). Please just look at my own comment at the link above, not the others, not from Unmannedtech either, as it contains incorrect mapping.. The mapping (even the mapping from airbot) is not correct!  (I had to find them myself, and these are the correct ones: gps on uart3, mag i2c1, sbus uart6, sport tx1). 
Regarding testing of sbus, go to the radio pane to check response from your transmitter, after you see your channels, then the modes. Let me know how it goes, regards

Reply:
atobiese,
wow what a mystery!  still not working.  I flashed in your patch.
For your patch, must serialrx_inverted = OFF or serialrx_inverted = ON?   I have tried both.  
I connected an oscilloscope to the SBUS pad on the flight controller.  I can see the pulse groups and when I move the throttle on taranis, I see changes in the pulses that correspond to the physical stick movement.  That verifies everything up to the FC input.  Uart6 Serial RX is on in the ports tab of iNav and everything else is disabled right now.   FC powers up in status tab will all green icons.  Still no activity in the Receiver tab or flight modes tab (all modes are disabled right now).   I'm starting to think the FC board might be bad.  Can't return it because I've soldered to it.
If you think of anything else, please let me know.  Otherwise, I'll have to trash this $35 FC because I don't have time to setup a build and get a debugger working to set breakpoints and troubleshoot.   :(